src/main/java/picard/analysis/CollectWgsMetrics.java

```diff
@@ -507,36 +521,38 @@ protected WgsMetrics generateWgsMetrics(final IntervalList intervals,
-                                            final long basesExcludedByPairing,
-                                            final long basesExcludedByBaseq,
-                                            final long basesExcludedByOverlap,
-                                            final long basesExcludedByCapping,
-                                            final int coverageCap,
-                                            final Histogram<Integer> baseQHistogram,
-                                            final int theoreticalHetSensitivitySampleSize) {
-        final double total              = depthHistogram.getSum();
-        final double totalWithExcludes  = total + basesExcludedByDupes + basesExcludedByMapq + basesExcludedByPairing + basesExcludedByBaseq + basesExcludedByOverlap + basesExcludedByCapping;
+                                          final Histogram<Integer> highQualityDepthHistogram,
+                                          final Histogram<Integer> unfilteredDepthHistogram,
+                                          final long basesExcludedByMapq,
+                                          final long basesExcludedByDupes,
+                                          final long basesExcludedByPairing,
+                                          final long basesExcludedByBaseq,
+                                          final long basesExcludedByOverlap,
+                                          final long basesExcludedByCapping,
+                                          final int coverageCap,
+                                          final Histogram<Integer> unfilteredBaseQHistogram,
+                                          final int theoreticalHetSensitivitySampleSize) {
+        final double total = highQualityDepthHistogram.getSum();
+        final double totalWithExcludes = total + basesExcludedByDupes + basesExcludedByMapq + basesExcludedByPairing + basesExcludedByBaseq + basesExcludedByOverlap + basesExcludedByCapping;
 
-        final double pctExcludedByMapq    = (0 == totalWithExcludes) ? 0 : (basesExcludedByMapq         / totalWithExcludes);
-        final double pctExcludedByDupes   = (0 == totalWithExcludes) ? 0 : (basesExcludedByDupes        / totalWithExcludes);
-        final double pctExcludedByPairing = (0 == totalWithExcludes) ? 0 : (basesExcludedByPairing      / totalWithExcludes);
-        final double pctExcludedByBaseq   = (0 == totalWithExcludes) ? 0 : (basesExcludedByBaseq        / totalWithExcludes);
-        final double pctExcludedByOverlap = (0 == totalWithExcludes) ? 0 : (basesExcludedByOverlap      / totalWithExcludes);
-        final double pctExcludedByCapping = (0 == totalWithExcludes) ? 0 : (basesExcludedByCapping      / totalWithExcludes);
-        final double pctTotal             = (0 == totalWithExcludes) ? 0 : ((totalWithExcludes - total) / totalWithExcludes);
+        final double pctExcludedByMapq = totalWithExcludes == 0 ? 0 : basesExcludedByMapq / totalWithExcludes;
```

Comment:
in general it makes it more difficult to review when there are many cosmetic changes...I'd prefer if you left working code untouched, unless there's a compelling reason. if you do have a stylistic change you'd like to do, I recommend putting it into a separate commit as it requires a different mind-set to review.